### DISCUSSION TITLE
SENSITIVITY OF stylus - Tablet Oscal Pad10 8GB/128GB

### DISCUSSION THREAD

**Discussion by ZdenekDanek** [oscal-club/community#38] [General]
Hello,
I have a Tablet Oscal Pad10 8GB/128GB and a normal stylus.
When I draw on the tablet in the drawing program, the line is jerky and reacts late, I can't even write. 
It works fin on mobile.
I suppose I should increase the sensitivity in the tablet settings so that there is no lag when drawing.

BUT HOW ??

Zdenek

**Comment by JonZeolla** [comment]
Hi @ZdenekDanek that's the wrong oscal (not related to this project). Best of luck getting your answer!

**Reply by ZdenekDanek** [reply]
Hi, can you please advise me where I should ask?

**Comment by xee5ch** [comment]
Thanks for the community member  assist, @JonZeolla. You're right. Good luck, @ZdenekDanek I will close this as it's unrelated to this OSCAL, not the mobile device you're asking about.
